ert_shared/ensemble_evaluator/prefect_ensemble/storage_driver.py

```diff
@@ -12,25 +12,29 @@ def storage_driver_factory(config, run_path):
 
 class _SharedDiskStorageDriver:
     def __init__(self, storage_path, run_path):
-        self._storage_path = storage_path
-        self._run_path = f"{run_path}"
+        self._storage_path = Path(storage_path)
+        self._run_path = Path(f"{run_path}")
```

Comment:
Why the format string?

Reply:
I noted several of those through the code, but at this point I did not want to go around fixing style thing in code somebody else wrote, while I am still the rookie in this project...

It is the kind of thing that should be detected/enforced by a linter. I noticed flake8 is run at some point in the CI, but only on ert3. Maybe we should consider making an issue to use a linter on this new code too?

Reply:
I think adding an issue on style checking more of the new modules would be good :) Feel free to create it...